src/platform/Tizen/ThreadStackManagerImpl.cpp

```diff
@@ -586,6 +586,17 @@ CHIP_ERROR ThreadStackManagerImpl::_ClearSrpHost(const char * aHostName)
     return CHIP_ERROR_NOT_IMPLEMENTED;
 }
 
+CHIP_ERROR ThreadStackManagerImpl::_DnsBrowse(const char * aServiceName, DnsBrowseCallback aCallback, void * aContext)
+{
+    return CHIP_ERROR_NOT_IMPLEMENTED;
+}
+
+CHIP_ERROR ThreadStackManagerImpl::_DnsResolve(const char * aServiceName, const char * aInstanceName, DnsResolveCallback aCallback,
+                                               void * aContext)
+{
+    return CHIP_ERROR_NOT_IMPLEMENTED;
```

Comment:
Am I reading correctly that the DNS functions are now directed here?  If so, is it intentional that we would throw `CHIP_ERROR_NOT_IMPLEMENTED`?  Is this then to allow for future, Thread-specific Tizen work in the sdk, or does this allow for some type of different override behavior in code consuming the Tizen platform?

Reply:
This is a shim for now. Proper Thread DNS implementations for Tizen require changes in Tizen Native API which right now is not available on tizen.org. I thought that such implementation (dummy one) will be better than nothing at all. Right now one will get a proper error message (not implemented) instead of running IP mDNS version of browse and resolve. I will submit for these functions later (remove, browse and resolve; and a fix for register call - right now it does not register TXT records).